ets-demo/etstool.py

```diff
@@ -311,6 +339,72 @@ def flake8(runtime, toolkit, environment):
     execute(commands, parameters)
 
 
+@cli.command(name="generate-bundles")
+def generate_bundles():
+    """ Generate application bundles for each supported platform.
+    """
+
+    for platform in PLATFORMS:
+        bundle_dir = os.path.join("bundle", platform)
+        if not os.path.exists(bundle_dir):
+            os.makedirs(bundle_dir)
+
+        bundle_file = os.path.join(
+            bundle_dir,
+            f"etsdemo-{APP_BUNDLE_VERSION}.bundle"
+        )
+
+        click.echo("Generating bundle {}".format(bundle_file))
+
+        edm_command = [
+            "edm",
+            "bundle",
+            "generate",
+            "--bundle-format",
+            "2.0",
+            "--platform",
+            platform,
+            "--version",
+            DEFAULT_RUNTIME,
+            "--output-file",
+            bundle_file,
+        ] + sorted(full_app_dependencies)
+        try:
+            subprocess.check_call(edm_command)
+        except subprocess.CalledProcessError:
+            click.echo("Failed to generate bundle {}".format(bundle_file))
+
+
+@cli.command(name="upload-bundles")
+@click.option('--repository', default=DEFAULT_REPOSITORY)
+def upload_bundles(repository):
+    """ Upload the generated bundles """
+    BUNDLES = []
+    for platform in PLATFORMS:
+        bundle_dir = os.path.join("bundle", platform)
+        bundle_file = os.path.join(
+            bundle_dir,
+            f"etsdemo_{platform.replace('-', '_')}-{APP_BUNDLE_VERSION}.bundle"
```

Comment:
The generated bundles don't have this name format anymore. It'll be useful to also refactor the bundle name into a module-level constant that is used both when generating and uploading the bundles.